api/src/main/java/org/openmrs/api/db/hibernate/search/LuceneAnalyzerFactory.java

```diff
@@ -58,6 +60,12 @@ public SearchMapping getSearchMapping() {
 			.filter(NGramFilterFactory.class)
 			.param("minGramSize", "2")
 			.param("maxGramSize", "20");
+		mapping.analyzerDef(LuceneAnalyzers.SOUNDEX_ANALYZER, StandardTokenizerFactory.class)
+			.filter(PhoneticFilterFactory.class)
+				.param("encoder", "Soundex")
+				.param("inject", "true");
+
+		
```

Comment:
Remove unnecessary blank line

Reply:
@sumangala028 thanks for your review. But as I pointed out above: The code of this PR does not work. So we should focus on making the code work and then after that is established we can focus on more formal and beautification issues.

if you want to participate helping to make this code work: https://talk.openmrs.org/t/soundex-search-in-lucenequeries/26068/13

Otherwise I think the energy could be rather used on different reviews.

Reply:
I see... thanks for pointing me to the discussion of this issue. Yes, the beautification issues can be pushed back till the code is fixed.